### PR TITLE
[WIP] Add MagnifyingGlass layer

### LINKED ISSUE #720: Magnifying glass
Although it doesn't seem to be maintained, it would be nice to have something like https://github.com/bbecquet/Leaflet.MagnifyingGlass

### PR DESCRIPTION
**Pull request by davidbrochart**
Fixes #720

### CODE REVIEW

File: js/src/layers/MagnifyingGlass.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  }
+
+  create_obj() {
+    this.layers = [];
+    this.layer_views = new widgets.ViewList(
+      this.add_layer_model,
+      this.remove_layer_view,
+      this
+    );
+    this.layer_views.update(this.model.get('layers'));
```
Comment (martinRenou) [inline js/src/layers/MagnifyingGlass.js:40]: This call is async, so the line after your list might be empty.

Comment (davidbrochart) [reply js/src/layers/MagnifyingGlass.js:40]: Good point, that was it, thanks!

**Comment by davidbrochart** [comment]
Still some work to do with attribute update.

**Comment by davidbrochart** [comment]
An improvement could be to listen to the mouse position on the front-end and to update the `lat_lng` trait accordingly.
Otherwise I think it is ready to be reviewed/merged.

**Comment by davidbrochart** [comment]
image

**Pull request merged**